[unify] Fixed issue in unify integration with module (#3354)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Fixed an issue where the **unify** command failed on integrations using an API a module, when not called from the content root.

Reviewer: ```suggestion
* Fixed an issue where **unify** failed on integrations using an API a module, when not called from the content root.
```

* **lint** will now fail on `LOG` usage in python files.
* Fixed an issue where **pre-commit** running when there was no change to the files
* Fixed an issue where errors in **validate** were logged as `info`.

> * Fixed an issue where the **unify** command failed to unify a integration with a module, in cases where the cwd was not the root of the content repository.

Reviewer: `with a module`, does it mean `which uses an API module`?

Author: Right.
`* Fixed an issue where the **unify** command failed to unify a integration with an API module, in cases where the cwd was not the root of the content repository.`
It's OK?

Reviewer: ```suggestion
* Fixed an issue where the **unify** command failed on integrations using an API a module, when not called from the content root.
```

* Updated the **format** command to use the content graph instead of the id_set file.
* Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
* Removed a redundant debug log on the `tools.get_file` function.

> * Fixed an issue where the **unify** command failed on integrations using an API a module, when not called from the content root.

Reviewer: move this to `unreleased` please
